### ISSUE TITLE
cool project

### ISSUE THREAD

**Issue by gumbobr0t** [0xSxZ/Veerus#3]

**Comment by phantasmq** [comment]
How do I remove the miner? It looks like there is a miner on my Pc, and it can only come from THIS. I checked directorys and scanned my Pc with antivirus (malwarebytes). How do I remove it???? there are always 30% more but it doesnt show the program in task manager

**Comment by dc203eec615967a8f6928725d2ad3c65** [comment]
why would you run a piece of malware on your own pc, are you stupid

**Comment by Dinosaurum** [comment]
> why would you run a piece of malware on your own pc, are you stupid

fucking no name random letter no profile picture ass second account, shut up, it creates a fucking discord logger/rat, I just tried to test it idiot bcuz I didn't check the r4t.exe which is obviously malware itself and not only the small discord rat as he says

**Comment by dc203eec615967a8f6928725d2ad3c65** [comment]
1. my username is a hash
2. github default profile pictures are heat so hop off
3. why would you run malware on your own pc

**Comment by Dinosaurum** [comment]
> 1. my username is a hash
> 2. github default profile pictures are heat so hop off
> 3. why would you run malware on your own pc

as I fucking said to test it you absolutely braindead idiot it creates a discord logger with many features like mienr that sends the stuff to your wallet and discord webhook but he also infected it with his own second webhook


just stfu mate, everybody knows that you just created your account probably (don't have the time to check) and thats why you have no pfp and defend lil kids probably skidding shit and putting a second webhook in

**Comment by dc203eec615967a8f6928725d2ad3c65** [comment]
yeah i did just create my account because i forgot my other accounts information lmao, and again, WHY. WOULD. YOU. RUN. MALWARE. ON. YOUR. OWN. COMPUTER.

**Comment by Dinosaurum** [comment]
> yeah i did just create my account because i forgot my other accounts information lmao, and again, WHY. WOULD. YOU. RUN. MALWARE. ON. YOUR. OWN. COMPUTER.

to test it

**Comment by dc203eec615967a8f6928725d2ad3c65** [comment]
theres this fancy thing called a virtual machine, you skids actually know nothing

**Comment by Dinosaurum** [comment]
I didn't know as first that he included his own malware, I wanted to build my own stub with my own webhook

**Comment by Dinosaurum** [comment]
> theres this fancy thing called a virtual machine, you skids actually know nothing

that was when I didn't know much about vms, atm I'm using https://tria.ge/

**Comment by dc203eec615967a8f6928725d2ad3c65** [comment]
just make a vm yourself why are you running off of another system
